src/styles/components/student-assignmentlist.styl

```diff
@@ -15,5 +15,11 @@
       margin-bottom: 1em
 
     .listitem > div:nth-child(1)
```

Comment:
I think Roger meant that instead of

```
    .listitem > div:nth-child(1)
      margin-right: 2em
    .listitem > div:nth-child(2)
      margin-right: 2em
    .listitem > div:nth-child(3)
      margin-right: 2em
    .listitem > div:nth-child(4)
      flex: 1 1 auto
      margin-right: 2em
```

it could be streamlined to

```
    .listitem > div:nth-child(1),
    .listitem > div:nth-child(2),
    .listitem > div:nth-child(3),
    .listitem > div:nth-child(4)
      margin-right: 2em
    .listitem > div:nth-child(4)
      flex: 1 1 auto
```

But no worries in any case since the styles will be removed.